data-serving/data-service/test/case.test.ts

```diff
@@ -42,23 +59,58 @@ describe('POST', () => {
             .send({ date: '2020-04-01', outcome: 'not-a-valid-outcome' })
             .expect(422, done);
     });
-    it('create with valid input should return 200 OK with json', (done) => {
-        request(app)
+    it('create with valid input should return 200 OK', async () => {
+        const inputDate = new Date('2020-04-01').toJSON();
+        const inputOutcome = 'recovered';
+        const res = await request(app)
             .post('/api/cases')
-            .send({ date: '2020-04-01', outcome: 'recovered' })
+            .send({ date: inputDate, outcome: inputOutcome })
             .expect('Content-Type', /json/)
-            .expect(200, done);
+            .expect(200);
+
+        expect(res.body.date).toEqual(inputDate);
+        expect(res.body.outcome).toEqual(inputOutcome);
     });
 });
 
 describe('PUT', () => {
-    it('update should return 200 OK', (done) => {
-        request(app).put('/api/cases/id').expect(200, done);
+    it('update present item should return 200 OK', async () => {
+        const id = '53cb6b9b4f4ddef1ad47f943';
+        const c = new Case({ _id: id, outcome: 'pending', date: '2020-04-21' });
+        await c.save();
```

Comment:
nit but save() actually returns the saved document with its id I believe
https://mongoosejs.com/docs/api/document.html#document_Document-save

perhaps it'd be best to use that returned ID instead of faking one ourselves?

Reply:
SG, updated.